staticaddr/withdraw/manager.go

```diff
@@ -430,20 +408,19 @@ func (m *Manager) handleWithdrawal(deposits []*deposit.Deposit,
 				deposit.Withdrawn,
 			)
 			if err != nil {
-				log.Errorf("Error transitioning deposits: %v",
+				log.Errorf("Error transitioning deposits: %w",
```

Comment:
%w should not be used in log. It will be logged as a wrong format.

I found more instances of this issue:

```
loop$ git grep -w log | grep %w
staticaddr/deposit/actions.go:                  log.Errorf("%v: %w", txLabel, err)
staticaddr/deposit/actions.go:          log.Debugf("error while sweeping expired deposit: %w", err)
staticaddr/deposit/manager.go:                          log.Errorf("Error sending OnStart event: %w",
staticaddr/deposit/manager.go:                  log.Errorf("Error sending OnStart event: %w", err)
staticaddr/loopin/actions.go:           log.Warnf("unable to decode sweep address: %w", err)
staticaddr/loopin/actions.go:                   log.Errorf("%v: %w", txLabel, err)
staticaddr/loopin/fsm.go:               log.Errorf("Error checking if loop-in is stored: %w", err)
staticaddr/loopin/fsm.go:               log.Errorf("Error updating loop-in: %w", err)
staticaddr/loopin/manager.go:                           log.Errorf("Error sending OnStart event: %w",
staticaddr/loopin/manager.go:                   log.Errorf("Error sending OnNewRequest event: %w", err)
staticaddr/withdraw/manager.go:                         log.Errorf("Error republishing withdrawals: %w",
staticaddr/withdraw/manager.go:                 log.Errorf("%v: %w", txLabel, err)
staticaddr/withdraw/manager.go:                         log.Errorf("Error transitioning deposits: %w",
staticaddr/withdraw/manager.go:                 log.Errorf("Error waiting for confirmation: %w", err)
staticaddr/withdraw/manager.go:                 log.Errorf("Error republishing withdrawal: %w", err)
```

Reply:
thanks for the hints!